apps/snitch_core/lib/core/domain/order/default_machine.ex

```diff
@@ -57,16 +57,16 @@ defmodule Snitch.Domain.Order.DefaultMachine do
 
   state_machine OrderSchema,
                 :state,
-                ~w(cart address payment processing rts shipping complete cancelled)a do
+                ~w(cart address payment delivery processing rts shipping complete cancelled)a do
     event(:add_addresses, %{from: [:cart], to: :address}, fn context ->
       context
       |> Transitions.associate_address()
       |> Transitions.compute_shipments()
       |> Transitions.persist_shipment()
     end)
 
-    event(:add_payment, %{from: [:address], to: :payment}, fn context ->
-      context
+    event(:process_packages, %{from: [:address], to: :delivery}, fn context ->
```

Comment:
`add_shipping_preference` is a better name for the event.

Reply:
👍